.github/workflows/release.yml

```diff
@@ -24,7 +25,7 @@ env:
   NUGET_PUBLISH_KEY: ${{ secrets.NUGET_PUBLISH_KEY }}
   NUGET_FEED_URL: https://api.nuget.org/v3/index.json
   PUBLISH_NUGET: true
-  # IF YOU NEED TO PUBLISH A PYPI PACKAGE THEN ENSURE AN PYPI_PASSWORD
+  # IF YOU NEED TO PUBLISH A PYPI PACKAGE THEN ENSURE AN PYPI_API_TOKEN
```

Comment:
The secret configured at `pulumiverse` organization level is `PYPI_PASSWORD`, not `PYPI_API_TOKEN`

Reply:
@ringods I'm puzzled 😮  That's the code in the PR branch.

![image](https://github.com/pulumiverse/pulumi-mssql/assets/14177833/7d757a0c-27d8-42fc-96a8-db35243beaf7)

and that's the code on my local machine:

![image](https://github.com/pulumiverse/pulumi-mssql/assets/14177833/c24a6492-e7d3-4949-889b-7660ff52413b)

To what code version you're referring your comment?

Reply:
@ringods AHA okay it's in the comment 😁

Reply:
@ringods Updated